.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -48,7 +48,7 @@ body:
     label: "Runtime information:"
     description: >
       Output from `import sys, numpy; print(numpy.__version__); print(sys.version)`
-      If you are running NumPy 1.24+, also show `print(numpy.show_runtime())`
+      If you are running NumPy 1.24+, also install `threadpoolctl` (e.g. with `pip` or `conda`) and include the output of `print(numpy.show_runtime())`.
```

Comment:
That line is getting rather long ...

Reply:
```suggestion
      If you are running NumPy 1.24+, also install `threadpoolctl` (e.g. with `pip` or `conda`),
      then include the output of `print(numpy.show_runtime())`.
```

Reply:
Should explicit newline characters be added? 
![image](https://github.com/numpy/numpy/assets/6570539/bb1a944e-932f-46f7-b20a-4b84ebc71e0d)

Reply:
Probably would render OK if the line breaks before this part.
```suggestion
      \n
      If you are running NumPy 1.24+, also install `threadpoolctl` (e.g. with
      `pip` or `conda`), then include the output of `print(numpy.show_runtime())`.
```

Reply:
See my latest push that uses a `|` multiline yaml string instead.

Reply:
Good to know.

Reply:
There is still a long, unbroken line.

Reply:
I decided to split this up into two text boxes: one required that asks for version info and one optional that asks for the output of `show_runtime()`. This lets me keep it a little more organized and not have the long line.